src/ray/raylet/worker_pool.h

```diff
@@ -702,146 +706,6 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
-  ///       | s3://bucket/my_module.zip   |           2            |
-  ///       +------------------------------------------------------+
-  ///   At the same time, we should remove URI reference for worker process because python
-  ///   worker only contains one worker instance:
-  ///       +-------------------------------------------------------------------+
-  ///       |      id              |          URIs                              |
-  ///       +----------------------+--------------------------------------------+
-  ///       | job-id-hex-A         | conda://{hash-A}                           |
-  ///       | worker-id-hex-A      | conda://{hash-A},s3://bucket/my_module.zip |
-  ///       +-------------------------------------------------------------------+
-  ///       +------------------------------------------------------+
-  ///       |      URI                    |          count         |
-  ///       +-----------------------------+------------------------+
-  ///       | conda://{hash-A}            |           2            |
-  ///       | s3://bucket/my_module.zip   |           1            |
-  ///       +------------------------------------------------------+
-  ///
-  ///   Next, when the actor is killed, remove URI reference for worker instance:
-  ///       +-------------------------------------------------------------------+
-  ///       |      id              |          URIs                              |
-  ///       +----------------------+--------------------------------------------+
-  ///       | job-id-hex-A         | conda://{hash-A}                           |
-  ///       +-------------------------------------------------------------------+
-  ///       +------------------------------------------------------+
-  ///       |      URI                    |          count         |
-  ///       +-----------------------------+------------------------+
-  ///       | conda://{hash-A}            |           1            |
-  ///       +------------------------------------------------------+
-
-  ///   Finally, when the job is finished, remove URI reference and got an empty reference
-  ///   table:
-  ///       +-------------------------------------------------------------------+
-  ///       |      id              |          URIs                              |
-  ///       +----------------------+--------------------------------------------+
-  ///       |                      |                                            |
-  ///       +-------------------------------------------------------------------+
-  ///       +------------------------------------------------------+
-  ///       |      URI                    |          count         |
-  ///       +-----------------------------+------------------------+
-  ///       |                             |                        |
-  ///       +------------------------------------------------------+
-  ///
-  ///  Now, we can delete the runtime env resources safely.
```

Comment:
Isn't there still value in keeping this? (Maybe moving it above the Increase/DecreaseURIReference methods)?  It's true that the actual adding and deleting of the references is now happening in the agent, but the AddURIReference and DeleteURIReference calls are still coming from this file.  So the information here is still accurate and useful, right?

Reply:
Yeah, I can add a doc to `Increase/DecreaseURIReference methods`. But it will be more simple which is entire different from this.

Reply:
Add the doc here https://github.com/ray-project/ray/pull/22828/files#diff-a88735bc6aa09dd8248a3e73081916b5096a79c4bb7de91f9875d2c5a84ef075R629